web/init/src/components/config_render/ConfigGroup.jsx

```diff
@@ -142,7 +142,8 @@ export default class ConfigGroup extends React.Component {
 
   render() {
     const { item } = this.props;
-    if (!this.isAtLeastOneItemVisible()) return null;
+    const hidden = item && item.when === "false";
```

Comment:
maybe order of operations here?
```suggestion
    const hidden = (item && (item.when === "false"));
```

Reply:
javascript would check if `item` exists first then evaluate the right hand expression `item.when === "false"`

Reply:
I know
Personally I feel the second is cleaner
But whatever code style we're using elsewhere is fine
It's JS, don't rely on my style sense lol